client/src/app/(onboarding)/signup/page.tsx

```diff
@@ -1,3 +1,171 @@
-export default function Page() {
-  return <h1>Signup placeholder</h1>;
+"use client";
+import { Button, TextField, Typography, Link, Divider } from "@mui/material";
+import "./signup.css";
+import { useState } from "react";
+/*
+This page will represent the signup page of the application,
+which include the signup with google button, email, confirm email, and password fields
+and logging in link.
+*/
+
+export default function SignupPage() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [confirmPassword, setConfirmPassword] = useState("");
+
+  const handleSubmit = () => {
+    if (password !== confirmPassword) {
+      console.log("Passwords do not match");
+      return;
+    }
+    // Handle signup logic here (e.g., call an API)
+    console.log("Email:", email);
+    console.log("Password:", password);
+  };
+
+  return (
+    <div className="main-container">
+      {/* Let's get started section */}
+      <Typography variant="h4" sx={{ marginBottom: "20px" }}>
+        Let’s Get Started.
+      </Typography>
+
+      {/* Sign in with Google Button */}
+      <Button
+        className="border-line"
```

Comment:
are you calling the .borderline from the global.css? If so it's "borderline" not "border-line"

Reply:
oh should it be className="borderline" then?

Reply:
all fixed